packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -177,37 +189,39 @@
     @include mdc-theme-dark(".mdc-text-field") {
       @include mdc-theme-prop(color, text-secondary-on-dark);
     }
+  }
 
-    &--float-above {
-      transform: translateY(-50%) scale(.75, .75);
-    }
+  .mdc-text-field__label--float-above {
+    transform: translateY(-50%) scale(.75, .75);
   }
 
   .mdc-text-field__label--float-above {
```

Comment:
nit: you can combine this with the `.mdc-text-field__label--float-above` above

Reply:
will fix in another PR after this